english/spoken_language_understanding.md

```diff
@@ -0,0 +1,42 @@
+# Spoken Language Understanding
```

Comment:
Adding a suggestion, since the `Spoken Language` indicates that this is voice, while this is not about voice or text - just language

```suggestion
# Conversational Understanding
```

Reply:
Yeah, that's a fair point. The usual research refer to this as Spoken Language Understanding for some reason. The alternative is intent classification and slot filling, which is a bit of a mouth full.

I think `Conversational Unerstanding` might imply there is some sort of dialogue relationship. What do you think of `Text Command Understanding`?

Reply:
That is good. 

Text commands are typically associated with text game commands or "/" commands in Chat/Slack. 

We can simplify, and make it more obvious: `Language Understanding` or `Natural Language Understanding`

This is also the named used by MSFT and Snips (`NLU-Benchmark`) both

What do you think?

Reply:
I think NLU is a bit too general as it is suppose to be an umbrella term for any generate language understanding.

Looking more over the papers for this topic, there really isn't a good term for the task. Maybe it's better to just making it explicit for now (i.e. `Intent Detection and Slot Filling`).

Reply:
Yup, explicit >> implicit in titles 

Thanks for considering the changes @edwardcqian